client/components/render-search-results.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+export default class RenderSearchResult extends React.Component {
+  render() {
+    const { results, addToLibrary } = this.props;
+
+    let author = '';
+    if (results.volumeInfo.authors !== undefined) {
+      author = results.volumeInfo.authors.join(', ');
+    }
+    let publishedYear = '';
+    if (results.volumeInfo.publishedDate !== undefined) {
+      publishedYear = results.volumeInfo.publishedDate.slice(0, 4);
+    }
+
+    let src = 'https://fivebooks.com/app/uploads/2010/09/no_book_cover.jpg';
```

Comment:
I recommend downloading this image to your server/public directory and linking it from there in case fivebooks moves or deletes it